Makefile.conf

```diff
@@ -5,25 +5,26 @@ ifeq ($(VERBOSE), false)
 endif
 TIMEOUT?=1m
 
-all: EXTLDFLAGS:=
-static: EXTLDFLAGS:=-extldflags "-static"
+STATIC_EXTLDFLAGS:='-extldflags=-static'
 
-.PHONY: all static
-all static: $(PROG)
+.PHONY: all install
+all install: .install
 
-.PHONY: all
-all: $(PROG)
+.PHONY: .install
+.install:
```

Comment:
no need to introduce `.install` target (just use `install`):

```
all: install
```

Reply:
Good idea. Done and pushed.